### PR TITLE
doc: change "master" to "main" in backport doc

### PR DESCRIPTION
**Pull request by yuvalif**
also fix the alias setting example

Signed-off-by: yuval Lifshitz <ylifshit@redhat.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by yuvalif** [comment]
@ljflores @smithfarm can you please review?

**Review by ljflores — APPROVED** [review]
We should probably eventually remove mentions of the "Stable Releases and Backports Team", since I don't think that's a thing anymore. But in terms of changing from master to main, this LGTM.

File: SubmittingPatches-backports.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ General principles
 ------------------
 
 To help the people who will review your backport, please state either in the
-backport PR, or in the backport tracker issue, or in the master tracker issue:
+backport PR, or in the backport tracker issue, or in the main tracker issue:
```
Comment (anthonyeleven) [inline SubmittingPatches-backports.rst:20]: To me `main` is even more generic than `master`, so I might make this `main branch tracker issue`.  I'd also be tempted to backtick every instance of `main` in the file but that'd be a lot of work.

**Review by anthonyeleven — APPROVED** [review]

File: SubmittingPatches-backports.rst (hunk 1 above)
Comment (yuvalif) [reply SubmittingPatches-backports.rst:20]: there are several places where the term "main tracker issue" appears. 
not clear if this is the "tracker issue of the main branch" or the "main tracker issue" (see line 140).
I prefer to keep this fix minimal.

Comment (yuvalif) [reply SubmittingPatches-backports.rst]: made the changes

**Comment by yuvalif** [comment]
jenkins test make check

**Comment by zdover23** [comment]
> jenkins test make check

As soon as this make check succeeds, I will create backport PRs for Quincy and Pacific, and I'll ask for your approval on them so that this change is live on docs.ceph.com.

**Comment by yuvalif** [comment]
jenkins test make check

**Pull request merged**